### PR TITLE
About Us update

### PR DESCRIPTION
**Pull request by cullam3n**
visually pleasing.  feel free to offer up feedback both on the visual aspect as well as the content

### CODE REVIEW

File: src/app/about/page.tsx [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
-      <h1 className="my-16 md:mb-12 text-5xl md:text-8xl font-bold tracking-tighter leading-tight md:pr-8">
+      <h1 className="my-16 md:mb-6 text-5xl md:text-8xl font-bold tracking-tighter leading-tight md:pr-8">
       Our Mission
 	    </h1>
 	    <div>
-		  Fighting for affordable, accessible, and reliable public transit for the continually growing Dallas area.  
-	    </div>
-	    <h1>
-        About Us
+		    <h2 className="text-3xl mt-12 mb-4 leading-snug leading-tight">
```
Comment (CurtisGreen) [inline src/app/about/page.tsx]: https://tailwindcss.com/docs/line-height

Leading snug & tight are overwriting the same thing

Comment (cullam3n) [reply src/app/about/page.tsx]: You're right, should be leading-tight everywhere

File: src/app/about/page.tsx [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
+        Fighting for affordable, accessible, and reliable public transit for the continually growing Dallas area.  
+	      </h2>
+      </div>
+	  <div>
+      <h1 className="my-8 md:mb-3 md:text-4xl font-bold leading-tight">
+      About Us
       </h1>
-      <div>
+    </div>
+      <div className="my-8 md:mb-6 text-2xl md:text-2xl leading-tight md:pr-8">
```
Comment (CurtisGreen) [inline src/app/about/page.tsx:17]: Divs are misaligned. If you can, setup your IDE to auto format on save. I think I have the project setup to use ESLint

**Review by CurtisGreen — CHANGES_REQUESTED** [review]
IMG_1673

Something is messed up on the bottom, need margin. You can look at other pages and copy the values to match

File: src/app/about/page.tsx (hunk 2 above)
Comment (cullam3n) [reply src/app/about/page.tsx:17]: fixed

**Comment by cullam3n** [comment]
> IMG_1673
> 
> Something is messed up on the bottom, need margin. You can look at other pages and copy the values to match

Other pages had 16px up/down margin with 12px bottom just for md, but because it's part of the list i just did 16px bottom and 12px bottom for md.  lmk if that's better

**Review by CurtisGreen — APPROVED** [review]

**Pull request merged**